docs/source/projections/eqc.rst

```diff
@@ -4,7 +4,95 @@
 Equidistant Cylindrical (Plate Caree)
```

Comment:
There is a point here. The plate carrée is just one case of equidistant cylindrical, isn't it ? So why using it on the title ...

done.